runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/DataflowPipelineTranslator.java

```diff
@@ -849,7 +849,8 @@ public void translate(CombineGroupedValues transform, TranslationContext context
 
             stepContext.addEncodingInput(fn.getAccumulatorCoder());
 
-            boolean isFnApi = DataflowRunner.useUnifiedWorker(context.getPipelineOptions());
+            List<String> experiments = context.getPipelineOptions().getExperiments();
```

Comment:
Assuming the ULR goes green, I think this piece can be omitted. This got pulled in but is not really related.